pkg/network/driver/common.go

```diff
@@ -136,21 +133,6 @@ func (h *NetworkUtilsHandler) AddrAdd(link netlink.Link, addr *netlink.Addr) err
 func (h *NetworkUtilsHandler) LinkSetMaster(link netlink.Link, master *netlink.Bridge) error {
 	return netlink.LinkSetMaster(link, master)
 }
-func (h *NetworkUtilsHandler) HasNatIptables(proto iptables.Protocol) bool {
```

Comment:
Just a thought: This changes things slightly I think. Typically nftables or iptables is there on a node, so we always meet this requirement in the past. I think now with only supporting nftables, you may consider surfacing a scheduling label by virt-handler which marks the node as having nftables, so that masquerade VMs don't go to nodes where nftables is not present?

May make the experience more smooth.